README.md

```diff
@@ -429,6 +431,8 @@ or an alternative with environment variable
 
 all variants will return your GCS buckets.
 
+`s5cmd` reads `.aws/credentials` to access Google Cloud Storage. Populate the `aws_access_key_ID` and `aws_secret_access_key` fields in `.aws/credentials` with an HMAC key created using the Google Cloud console or `gsutil`.
```

Comment:
```suggestion
`s5cmd` reads `.aws/credentials` to access Google Cloud Storage. Populate the `aws_access_key_id` and `aws_secret_access_key` fields in `.aws/credentials` with an HMAC key created using the Google Cloud console or `gsutil`.
```

----------


If there's an official documentation from GCS a link would be nice.

Reply:
> https://github.com/peak/s5cmd/pull/377#discussion_r744246944 If there's an official documentation from GCS a link would be nice.

@pjgo I would prefer reference to an official documentation opposed to this and look like the sentence is duplicated in both section. 

I think there should be a _reference to the GCS documentation_ _only in GCS section_. If not we might as well just close this as GCS is no different than the other S3 compatible services from `s5cmd`'s point of view.